email_mapping/email_mapping.py

```diff
@@ -11,18 +11,21 @@
 import pyarrow as pa
 import pyarrow.parquet as pq
 import s3fs
+import sys
 
-RotatingFileHandler("mit_open_etl.log").push_application()
-log = Logger("mit_open_etl")
+app_name = sys.argv[1]
+
+RotatingFileHandler("{}_etl.log".format(app_name)).push_application()
+log = Logger("{}_etl".format(app_name))
```

Comment:
Should we go with [`task_name`](https://github.com/mitodl/salt-ops/blob/master/salt/etl/init.sls#L1) or change the other one to app_name?

Reply:
Nevermind. While reviewing the other PR, I see the difference between task and app name